src/screens/ChildrenScreen.js

```diff
@@ -1,158 +1,137 @@
-    },
-    {
-      type: "Covid-19",
-      day: 1,
-      month: "January",
-      year: 2021,
-      note: "Next does in date: 30-Mar-2021",
-    },
-    {
-      type: "Covid-19",
-      day: 1,
-      month: "January",
-      year: 2021,
-      note: "Next does in date: 30-Mar-2021",
-    },
-    {
-      type: "Covid-19",
-      day: 1,
-      month: "January",
-      year: 2021,
-      note: "Next does in date: 30-Mar-2021",
-    },
-    {
-      type: "Covid-19",
-      day: 1,
-      month: "January",
-      year: 2021,
-      note: "Next does in date: 30-Mar-2021",
-    },
-    {
-      type: "Covid-19",
-      day: 1,
-      month: "January",
-      year: 2021,
-      note: "Next does in date: 30-Mar-2021",
-    },
-    {
-      type: "Covid-19",
-      day: 1,
-      month: "January",
-      year: 2021,
-      note: "Next does in date: 30-Mar-2021",
-    },
-  ];
+const users = [
+  {
+    id: 1,
+    name: "JasperMattews",
+    birth: "Jan 15th 2020",
+    latest: "2020-03-20",
+    avatar: "https://source.unsplash.com/user/nicoleknipes",
+  },
+  {
+    id: 2,
+    name: "JasperMattews",
+    birth: "Jan 15th 2020",
+    latest: "2020-03-20",
+    avatar: "https://source.unsplash.com/user/nicoleknipes",
+  },
+  {
+    id: 3,
+    name: "JasperMattews",
+    birth: "Jan 15th 2020",
+    latest: "2020-03-20",
+    avatar: "https://source.unsplash.com/user/nicoleknipes",
+  },
+];
 
-  renderHistories = () => {
-    return (
-      <View>
-        {historyArr.map((data, index) => {
+const ChildrenScreen = ({ navigation }) => {
+  return (
+    <View style={styles.mainContainer}>
+      <View style={styles.header}>
+        {users.map((u) => {
           return (
-            <View style={styles.container}>
-              <View style={styles.listStyle}>
-                <TouchableOpacity
-                  onPress={() =>
-                    navigation.navigate("ScheduleScreen", {
-                      name: "ScheduleScreen",
-                    })
-                  }
-                >
-                  <Text style={styles.type}>{data.type}</Text>
-                  <Text style={styles.date}>
-                    <Ionicons
-                      name="calendar"
-                      size={15}
-                      color={colors.darkGrey}
-                    ></Ionicons>
-                    {data.day} {data.month}, {data.year}
-                  </Text>
-                  <Text numberOfLines={1} style={styles.note}>
-                    {data.note}
-                  </Text>
-                </TouchableOpacity>
-                <Ionicons
-                  name="ios-arrow-forward"
-                  size={30}
-                  color={colors.lightBlue}
-                  style={styles.arrowIcon}
-                ></Ionicons>
+            <View key={u.id}>
+              <View style={styles.container}>
+                {/* <Image source={{ uri:u.avatar }} /> */}
```

Comment:
Please use TouchableOpacity and make it to navigate to ScheduleScreen when user click a item.